src/error.rs

```diff
@@ -7,6 +7,10 @@ use std::sync::mpsc::RecvError;
 /// Categories of Warpgrapher errors.
 #[derive(Debug)]
 pub enum ErrorKind {
+    /// Returned by the `From` implementation for a `actix_http::error::Error` object.
+    /// Conversion from `actix_http::error::Error`.
+    ActixHttpError(String),
```

Comment:
Shouldn't this go away if we're stripping out the dependency on a specific HTTP framework?  If it does need to stay, I'd recommend cleaning up the error definition comment.  I can't tell from this description why it would be thrown or what it would mean.

Reply:
This has been removed. I believe I added this at one point during development because I abstracted away the enginification away from the resolver tests and am still using Actix there and needed to to propagate out the error.